pkgs/development/libraries/tbb/default.nix

```diff
@@ -1,36 +1,26 @@
-{ stdenv, fetchFromGitHub, fixDarwinDylibNames, compiler ? if stdenv.cc.isClang then "clang" else null, stdver ? null }:
```

Comment:
I don't think specifying a compiler is needed anymore.
Otherwise using `overrideAttrs` to set cmakeFlags should be easy enough.